draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -640,28 +596,30 @@ The coherent pluggable YANG data model is augmented with a list of new capabilit
 The {{solution-vs-cap-attributes}} shows an example of new capability attribute "CC" whose min and max values are 1.1 and 3.3, respectively.
 
 ~~~~ 
- +--ro vendor-specific-capability-attribute-list* [cap-attribute-id]
+ +--ro opaque-capability-attribute-list* [cap-attribute-id]
      +--ro cap-attribute-id              uint32
      +--ro cap-attribute-name            string
      +--ro cap-min-value                 decimal64
      +--ro cap-max-value                 decimal64
 
-<vendor-specific-cap-attribute-list xmlns="urn:example:cc">
+<opaque-capability-attribute-list xmlns="urn:example:cc">
   <cap-attribute-id> 1 </cap-attribute-id >
   <cap-attribute-name> CC </cap-attribute-name>
   <cap-min-value> 1.1 </cap-min-value>
   <cap-max-value> 3.3 </cap-max-value>
-</vendor-specific-cap-attribute-list>
+</opaque-capability-attribute-list>
 ~~~~
-{: #solution-vs-cap-attributes title="Support of Vendor-Specific Capability Attributes"}
+{: #solution-vs-cap-attributes title="Support of Opaque Capability Attributes"}
 
-To support the vendor-specific configuration attributes, there are a few potential solutions which are discussed below.
+### Support of Opaque Secret Capability Attributes
+
+to be added
 
-{: #vendor-specfic-config-solution-1}
+{: #opaque-config-solution-1}
 
-### Vendor-Specific Configuration Attributes (Solution-1)
+### Support of Opaque Configuration Attributes (Solution-1)
 
-This approach is the simplest solution, as it requires no interpretation by the host platform. The coherent pluggable YANG data model is augmented with a list that directly maps the values of new configuration attributes to the corresponding memory-map locations on the pluggable device. In this solution, the memory-map locations must be known to the operator, potentially provided in the Coherent Pluggable Manifest. The {{solution-1}} illustrates the coherent pluggable YANG data model, which has been augmented with the following information:
+To support the opaque configuration attributes, there are a few potential solutions which are discussed below. This approach is the simplest solution, as it requires no interpretation by the host platform. The coherent pluggable YANG data model is augmented with a list that directly maps the values of new configuration attributes to the corresponding memory-map locations on the pluggable device. In this solution, the memory-map locations must be known to the operator, potentially provided in the Coherent Pluggable Repository. The {{solution-1}} illustrates the coherent pluggable YANG data model, which has been augmented with the following information:
```

Comment:
it may look simple but is impossible to debug. We should leave register maps to other SDOs